bangazonapi/views/profile.py

```diff
@@ -277,10 +277,22 @@ def favoritesellers(self, request):
         customer = Customer.objects.get(user=request.auth.user)
```

Comment:
It would be better to set this function up like the some of the other ones that have a multiple methods with an `if request.method === get`. 277-278 and 290-293 would be in this if statement